tests/cbor_decode/test5_strange/src/main.c

```diff
@@ -43,6 +43,21 @@ void test_numbers(void)
 }
 
 
+void test_numbers2(void)
+{
+	uint32_t decode_len = 0xFFFFFFFF;
+	const uint8_t payload_numbers2[] = {
+		0x81, // List start
+			0x1A, 0x00, 0x12, 0x34, 0x56, // 0x123456
```

Comment:
Minor: formatting is a bit off. Is this intentional or a linter result?

Reply:
It's to demonstrate the nesting in the data. I've done it in the other tests as well.